measures/iama.yaml

```
@@ -0,0 +1,130 @@
+systemcard_path: .assessments[]             # append iama.yaml to 'assessments' list in system card 
+
+instrument:
+  instrument_name: "Impact Assessment Mensenrechten en Algoritmes (IAMA)"
```

I think the key "instrument_name" has redundancy because it is already part of the object "instrument".
```suggestion
  name: "Impact Assessment Mensenrechten en Algoritmes (IAMA)"
```